### PR TITLE
React forms

### PR DESCRIPTION
**Pull request by SuduguTejasvi**

### CODE REVIEW

**Comment by SuduguTejasvi** [comment]
image

File: package-lock.json
Comment (MajaharZemoso) [inline package-lock.json:1]: You can add this file in .gitignore

File: src/App.tsx [hunk 1]
```diff
@@ -1,21 +1,33 @@
...
-import CalculatorPage from './pages/index';
+import { ThemeProvider } from '@mui/material/styles';
+import { CssBaseline, IconButton } from '@mui/material';
+import { lightTheme, darkTheme } from '../src/Themes/index';
+import { Brightness4, Brightness7 } from '@mui/icons-material';
+import CardLayout from './components/Template';
+export const isDark = createContext(false);
+
 function App() {
+  const [isDarkMode, setIsDarkMode] = useState(false);
```
Comment (MajaharZemoso) [inline src/App.tsx:11]: add type here

File: src/components/organisms/Card/index.tsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+import { adminB, adminW } from "../../../utils/constants/index";
+import Icon from "../../atoms/Icons";
+import { CustomGridRight } from "./index.styles";
+import { isDark } from "../../../App";
+import { darkTheme } from "../../../Themes/index";
+import { ToastContainer, toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+
+export default function Form() {
+  const isDarkMode = useContext(isDark);
```
Comment (MajaharZemoso) [inline src/components/organisms/Card/index.tsx:15]: +1

File: src/components/organisms/Card/index.tsx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+          draggable: true,
+          progress: undefined,
+          theme: "colored"
+          });
+      }
+    }
+  };
+
+  return (
+   <>
```
Comment (MajaharZemoso) [inline src/components/organisms/Card/index.tsx:54]: Don't use empty fragments

File: package-lock.json
Comment (SuduguTejasvi) [reply package-lock.json:1]: addressed it

File: src/App.tsx (hunk 1 above)
Comment (SuduguTejasvi) [reply src/App.tsx]: addressed it

File: src/components/organisms/Card/index.tsx (hunk 2 above)
Comment (SuduguTejasvi) [reply src/components/organisms/Card/index.tsx]: addressed it

File: src/components/organisms/Card/index.tsx (hunk 3 above)
Comment (SuduguTejasvi) [reply src/components/organisms/Card/index.tsx]: addressed it

**Review by MajaharZemoso — APPROVED** [review]

**Pull request merged**